File: python/oddities/oddities.py
def main():
    numOfIn = int(input())
    for x in range(numOfIn):
        userIn = int(input())
        if (abs(userIn) % 2 == 0):
            print(userIn, " is even")
        else:
            print(userIn, " is odd")

if __name__ == "__main__":
    main()

File: cpp/oddities/oddities.cpp
#include <iostream>

using namespace std;

int main(){
    int numOfIn, userIn;
    cin >> numOfIn;
    for (int i = 0; i < numOfIn; i++){
        cin >> userIn;
        if (abs(userIn) % 2 == 0)
            cout << userIn << " is even" << endl;
        else
            cout << userIn << " is odd" << endl;
    }
    return 0;
}